### ISSUE TITLE
comment out in .env is not recoginized

### ISSUE THREAD

**Issue by dumblepy** [euantorano/dotenv.nim#5]
It happens in v2.0.0

.env
```env
HOGE=true # ture or false
```

nim
```nim
import os, dotenv

load(getCurrentDir(), ".env")
getEnv("HOGE") == "true # ture or false"
```
but expected `getEnv(HOGE)` is `"ture"`

**Comment by euantorano** [comment]
Ah, good catch. I'll add a test case and fix for this in a few moments. I think the previous tests only cover quoted values.

**Comment by euantorano** [comment]
I've just released v2.0.1, which should resolve this issue and also adds support for single quoted values.
